### PR TITLE
Expose FIXED_EXPONENT from C++

### PR DESCRIPTION
**Pull request by badisa**
* Moves fixed->float and float->fixed to timemachine.lib.fixed_point
* Follow up from #1090 (https://github.com/proteneer/timemachine/pull/1090#discussion_r1269982754)

### CODE REVIEW

File: timemachine/lib/fixed_point.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import jax.numpy as jnp
+
+from timemachine.lib import custom_ops
+
+
+def FIXED_TO_FLOAT(v) -> jnp.float64:
+    return jnp.float64(jnp.int64(jnp.uint64(v))) / custom_ops.FIXED_EXPONENT
+
+
+def FLOAT_TO_FIXED(v) -> jnp.uint64:
```
Comment (proteneer) [inline timemachine/lib/fixed_point.py:10]: Don't we go through `int128` for energies in the C++ layer now?

Comment (proteneer) [reply timemachine/lib/fixed_point.py:10]: Thanks - makes sense to me.

File: timemachine/lib/fixed_point.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import jax.numpy as jnp
+
+from timemachine.lib import custom_ops
+
+
+def FIXED_TO_FLOAT(v) -> jnp.float64:
```
[the remark this answers is not in the document]
Comment (mcwitt) [reply timemachine/lib/fixed_point.py:6]: nit: should input type be `int`?

**Review by mcwitt — APPROVED** [review]
Nice! Lgtm with a few minor nits.

File: timemachine/lib/fixed_point.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (badisa) [reply timemachine/lib/fixed_point.py]: Added type hints and snake case in [1575a85](https://github.com/proteneer/timemachine/pull/1106/commits/1575a856217428071b66c3d2198afe9e95711794)

**Pull request merged**